Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -210,6 +214,14 @@ public class FieldNpc : Actor<Npc> {
             foreach (string tag in Value.Metadata.Basic.MainTags) {
                 player.Session.ConditionUpdate(ConditionType.npc_race, codeString: tag);
             }
-        }
+		}
+	}
+	
+    public void SetAi(string name) {
```

Comment:
you probably want a `[MemberNotNull(nameof(AiMetadata))]` here, might be a way to avoid marking `AiMetadata` as nullable

Reply:
Not all npcs have ai scripts on em. Nullable is required for AiMetadata
`11200003.xml: <aiInfo path="" />`

Reply:
Makes sense, what about `MemberNotNullWhen` instead?